docs/articles/user-guide/cli-commands.md

```diff
@@ -23,31 +23,39 @@ These Salesforce CLI commands are available in Salesforce Extensions for VS Code
 - `force:apex:test:run --resultformat human ...`: **SFDX: Invoke Apex Tests...**
 - `force:apex:trigger:create ...`: **SFDX: Create Apex Trigger**
 - `force:auth:logout --all --noprompt`: **SFDX: Log Out from All Authorized Orgs**
-- `force:auth:web:login --setdefaultdevhubusername`: **SFDX: Authorize a Dev Hub**
+- `force:auth:web:login --setdefaultdevhubusername`, `force:auth:device:login` (in remote development): **SFDX: Authorize a Dev Hub**
 - `force:config:list`: **SFDX: List All Config Variables**
+- `force:config:set --isvebuggerSid --isvDebuggerUrl --instanceUrl`: **Create and Set Up Project for ISV Debugging** (step 2)
+- `force:data:record:create --sobjecttype --values --usetoolingapi`: **Turn On Apex Debug Log for Replay Debugger**
+- `force:data:record:delete --sobjecttype --sobjectid --usetoolingapi`: **Turn Off Apex Debug Log for Replay Debugger**
 - `force:data:soql:query`: **SFDX: Execute SOQL Query with Currently Selected Text**
 - `force:data:soql:query ...`: **SFDX: Execute SOQL Query...**
 - `force:lightning:app:create ...`: **SFDX: Create Lightning App**
-- `force:lightning:component:create ...`: **SFDX: Create Lightning Component**
+- `force:lightning:component:create ...`: **SFDX: Create Lightning Web Component**
 - `force:lightning:event:create ...`: **SFDX: Create Lightning Event**
 - `force:lightning:interface:create ...`: **SFDX: Create Lightning Interface**
+- `force:mdapi:retrieve --retrievetargetdir --unpacked --targetusername`: **Create and Set Up Project for ISV Debugging** (step 4)
+- `force:mdapi:convert --rooter --outputdir force-app`: **Create and Set Up Project for ISV Debugging** (step 5)
+- `force:package:installed:list --targetusername`: **Create and Set Up Project for ISV Debugging** (step 6)
 - `force:org:create --setdefaultusername ...`: **SFDX: Create a Default Scratch Org**
-- `force:org:display`: **SFDX: Display Org Details for Default Scratch Org**
+- `force:org:display`: **SFDX: Display Org Details for Default Org**
 - `force:org:display --targetusername ...`: **SFDX: Display Org Details...**
 - `force:org:open`: **SFDX: Open Default Org**
 - `force:project:create --template standard ...`: **SFDX: Create Project**
 - `force:project:create --template standard --manifest ...`: **SFDX: Create Project with Manifest**
-- `force:source:delete`: **SFDX: Delete from Project and Org** (beta)
-- `force:source:deploy`: **SFDX: Deploy Source to Org** (beta)
-- `force:source:deploy --manifest ...`: **SFDX: Deploy Source in Manifest to Org** (beta)
+- `force:source:delete`: **SFDX: Delete from Project and Org**
+- `force:source:deploy`: **SFDX: Deploy Source to Org**
```

Comment:
We're missing `SFDX: Deploy This Source to Org`

Reply:
Added the command